recipes/pyspark_dist_explore/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pyspark_dist_explore" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1ce5316c5fbb75ff23623d3b12b22ac25c43eeb7d23ab25bc94532873311a0da
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
```

Comment:
Is numpy really required here? ...

Reply:
@dbast Thanks for looking at this PR. I've added the Licence file and added the path format as mentioned above. 

The setup.py file mentions numpy and the module imports numpy. I've tried to follow the documentation found here: https://conda-forge.org/docs/meta.html#building-against-numpy

I've removed the numpy import you've mentioned anyway, hope it is correct now.

Reply:
The package is **using** numpy... your are not **building** it against numpy (that is what your referred docu link is about) , e.g. it does not contain C code that is compiled against numpy. Otherwise if there would be non python code to compile also `noarch: python` would not work. That is why numpy is not required as host requirement and you can also just use `numpy` instead of `{{ pin_compatible('numpy') }}` as run dependency ... Not that important, but you could also bring the host/run requirements in alphabetical order ;)

Reply:
Thanks, that explains it! Do you have an idea why AppVeyor is failing? Because the PR is out of date compared to the master?